index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse mobile project</title>
+    <link rel="stylesheet" href="./css/style.css">
+</head>
+<body>
+    <header>
+        <section class="logo-menubar">
+            <h4 class="logo">
+                My Logo
+            </h4>
+            <nav>
+                <img src="./img/Normal Button/Tertiary/Icons/Icon - Menu.svg" class="hamb-icon" alt="menu-bar">
+            </nav>
+        </section>
+    </header>
+    <section class="presentation-part">
+        <h1>I'm Ivonne. Glad to see you!</h1>
+        <p class="presentation-paragraph">I am a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+        </p>
+        <section class="socialnetwork-part">
+           <p class="socialtext-part">
+            LET'S CONNECT
+           </p>
+           <img src="./img/social-media-icons.svg" alt="social-network icons">
```

Comment:
- [ ] The project contains two sections, the toolbar (or header):Toolbar, the headline section (right after the header): kindly consider grouping the headline section into one `section tag`

Reply:
Dear @ademibowale 
I ordered the project according your suggestions,
thank you for this feedback to my project 
I appreciate it,
Best regards,